modules/dnn/src/cuda/add_naive.cu

```diff
@@ -0,0 +1,47 @@
+#include <cuda_runtime.h>
+#include <float.h>
+#include "conv_naive.hpp"
+#include <cudnn.h>
+#include <cuda_fp16.h>
+#include <cuda_bf16.h>
+#include <cstdio>
+
+namespace cv { namespace dnn { namespace cuda {
+
+void add2(
```

Comment:
looks like this implementation is suitable for all binary arithmetic operations (+, -, *, ...). Can we give it more generic name and reuse it?